WowCharComparerWebApp/Views/Register/PerformUserRegister.cshtml

```diff
@@ -2,4 +2,4 @@
     WowCharComparerWebApp.Models.Internal.User registeredUser = ViewData.Model as WowCharComparerWebApp.Models.Internal.User;
 }
 
-<b>Thanks for registration @registeredUser.Nickname</b>
\ No newline at end of file
+<b>Thanks for signing up. Verification mail has been sended.  @registeredUser.Nickname</b>
```

Comment:
Maybe it is worth to add name of the email instead of nickname, or even both.
Something like 
`Thanks for singing up @registeredUser.Nickname! Verification mail has been send on adress @registeredUser.Email`

Reply:
Improved to be more friendly.